distributions/otelcol-k8s/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.16 as certs
+RUN apk --update add ca-certificates
+
+FROM scratch
+
+ARG USER_UID=10001
+USER ${USER_UID}
+
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY --chmod=755 otelcol-k8s /otelcol-k8s
+COPY config.yaml /etc/otelcol-k8s/config.yaml
+ENTRYPOINT ["/otelcol-k8s"]
+CMD ["--config", "/etc/otelcol-k8s/config.yaml"]
+EXPOSE 4317 4318 55678 55679 6831 14268 14250 9411
```

Comment:
Can you double check those ports?

Reply:
@jpkrohling these ports correspond with:

- `4137` and `4318`: OTLP
- `55678`: opencensus
- `55679`: zpages
- `6831`, `14268`, and `14250`: jaeger
- `9411`: zipkin

Reply:
Maybe it's worth adding a comment with this information to the Dockerfile?

Reply:
I'm fine resolving this, as this list of components match the current state of the PR. If we end up removing opencensus/zipkin/jaeger, the ports would be removed.